cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -233,7 +250,12 @@ func (w *wizard) vote() {
 
 	result := httpPostWithParams(fmt.Sprintf("http://%v:%v/vote", w.host, w.httpPort), data)
 
-	fmt.Printf("\n%v\n", result)
+	if result == "" {
```

Comment:
`result` will never be empty - it will contain error string

Reply:
Upon error `httpPostWithParams` will simply log the error and return an empty string: https://github.com/livepeer/go-livepeer/blob/1d3fd984e771d90ad534190ad0d964803cf7028a/cmd/livepeer_cli/wizard.go#L286

Reply:
It will do this in case it can't contact webserver. But if HTTP request was successful, but status code is not 2xx - it is not considered error, so `httpPostWithParams` will not log anything and will return body of request (which is in our case is a error message provided by the `respondWithError`) as a result

Reply:
Trying to finish up this PR. So there seems to be an issue here where we return a string regardless of the status code and it doesn't seem in the current structure of things there is an easy way to differentiate between a success and a failure as the error messages aren't prefixed with a status code or "Error: ..."

Reply:
Yes, best way would be to change `httpPostWithParams` to return `(string, bool)`, something like that:
```go
return string(result), resp.StatusCode >= 200 && resp.StatusCode < 300
```

Reply:
fixed in 4cb406d and cd54636